cityofaustin/vision-zero
# Create ETL that joins crashes and Moped components

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/19453

## Testing

**Steps to test:**
First fill out the env_template for Hasura, from 1pass. Make sure you use Moped **production** and VZ **staging** credentials 

Note, if you are on Apple Silicon you may need to add `--platform linux/amd64` to get GDAL to install correctly.

```
docker build . -t atddocker/vz-moped-join
```
```
docker run -it --env-file env_file atddocker/vz-moped-join /bin/bash
python moped_project_components_spatial_join.py
```

The logging should be pretty self explanatory to see if things succeeded or not.

You can also then check if `moped_component_crashes` was populated with the lookup table. (maybe delete everything in that table yourself to double check if the script did anything).

---

#### Ship list
